vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
+
+* Updated Keycloak to version 15.
+
+### Infrastructure
+
+* Includes a massive rework of the internal dependency management system wrt the concept of ProvidedCapability.
+* Added support for OKD4.8 and kube 1.21.
+* Added ComponentManager and Keycloak hardening.
+* Tuned Entando Platform timeouts and memory limits.
+* Moved from a HELM based deployment process to a pure CustomResource based deployment and installation process.
+* Installation parameters considered by the operator all moved to the entando-operator-config ConfigMap.
+* Better support for cloud DBMS.
+* Updated libraries (Jetty, Keycloak, Spring) and the entire compatibility matrix.
+* Dropped support for suffix-mode installation. Only single-hostname is supported now.
+* Temporarily dropped support for cluster-level operator installation.
+* Removed EntandoClusterInfrastructure and EntandoCompositeApp CR.
+* k8s-service is now part of the base Entando infrastructure and therefore installed alongside the operator.
+* ComponentManager now communicates with k8s-service using service-to-service authentication, and strictly within the cluster network.
+* Entando Operator deployment is now simply called entando-operator.
+* k8s-service deployment dropped the application qualifier and is now simply called entando-k8s-service.
+* Fixed an intermittent issue with the operator that prevented installing new bundles until the operator was restarted.
+* Updated all the Entando base images for reliability and security.
+* Replaced the centos images to rocky images and rebased the derived images.
+* Updated the auto provisioned PostgreSQL image to version 14.
+* MySQL and PostgreSQL are now served by customized and updated entando-mysql and entando-postgres images.
+
+### Entando Bundler
 
-Entando implements the SQL versioning tool Liquibase to track, manage and deploy updates to databases. Refer to [Liquibase Migration](../reference/databases.md#liquibase-migration) to learn how the App Engine automates database schema management and modifications.
+* It’s now possible to extract protected assets.
+* Fixed several bugs in the extraction of pages and widgets.
+* Added support for categories.
+* Error handling has been improved.
+* It's now possible to generate the so-called system-level bundles (see ECR).
+* Extract/download optimized to only touch the relevant resources.
+* Libraries updated for reliability and security.
+
+### EOL Kubernetes Versions
+
+Entando7 drops official support for for OS3.11 and Kubernetes versions prior to 1.20.
+
+### Google Kubernetes Engine (GKE)
+
+* Postponed support to 7.0.1.
+
+### Azure Kubernetes Service (AKS)
+
+* Postponed support to 7.0.1.
+
+## Breaking Changes
+
+* The internal PostgreSQL version has been updated to version 14. Bundles that rely on the auto provisioning of PostgreSQL may (or may not) be affected.
+
+## Known Issues
+
+### ECG
+
+* The default "details" widget generated for an entity doesn't work out of the box.
```

Comment:
@w-caffiero-entando is there a ticket for this? This is included in the tutorial and in our training. CC: @GermanoGiudici

Reply:
@w-caffiero-entando Several of the Known Issues here appear to have come from the 6.3.2 release notes - https://developer.entando.com/v6.3.2/docs/releases/#known-issues. I think that could be a source of confusion here so if we can clean up some of those, it would be helpful.